patches/tModLoader/Terraria.ModLoader.Recipes/RecipeBuilder.cs

```diff
@@ -0,0 +1,184 @@
+﻿using System;
+using Terraria;
+using Terraria.ModLoader;
+using Terraria.ModLoader.Exceptions;
+
+namespace Terraria.ModLoader.Recipes
+{
+	public class RecipeBuilder
+	{
+		private readonly ModRecipe _recipe;
+
+
+		/// <summary>Creates a new instance with no ingredients and no result.</summary>
+		/// <seealso cref="Requires(int,int)"/>
+		/// <seealso cref="Build"/>
+		public RecipeBuilder()
```

Comment:
It also doesn't initialize the recipe, remove this.

Reply:
Will do.